tests/test-rest-attachments-controller.php

```diff
@@ -20,6 +20,17 @@ public function setUp() {
 		$this->contributor_id = $this->factory->user->create( array(
 			'role' => 'contributor',
 		) );
+		//matches core capabilities for subscriber role and adds upload_files
+		_x( 'File upload role', 'User role' );
```

Comment:
well 'File upload role' in line 25 should be translated but doesn't need to be for tests I guess